src/server.ts

```diff
@@ -47,10 +57,22 @@ export async function startPluginsServer(config: PluginsServerConfig): Promise<v
     })
 
     // every 5 sec set a @posthog-plugin-server/ping redis key
-    schedule.scheduleJob('*/5 * * * * *', function () {
+    const job = schedule.scheduleJob('*/5 * * * * *', function () {
         redis.set('@posthog-plugin-server/ping', new Date().toISOString())
         redis.expire('@posthog-plugin-server/ping', 60)
     })
-
     console.info(`✅ Started posthog-plugin-server v${version}!`)
+
+    for (const signal of ['SIGINT', 'SIGTERM', 'SIGHUP']) {
```

Comment:
Actually could we put this in its own function like `setUpExitHandling(handler: (() => (void | Promise<void>))): void` ? Because the web server does identical `process.on(signal, handler)` with SIGINT, SIGTERM and SIGHUP and

Reply:
Ah, nvm, that's integrated!

Reply:
I actually moved `stopWebServer` also here.